bin/index.js

```diff
@@ -192,15 +222,15 @@ var debugLog = (log) => {
         }
 
         if (options.receivePort)
-            app.listen(options.receivePort, listener);
```

Comment:
well done🙏
can you write a function for this? it duplicates four times in the code 
after that you can merge it, also it’s good to add a TODO about creating a self-signed certificate if no cert and key provided.

Reply:
if you add
- support ssl
in the README it would be so nice

Reply:
> well done🙏
> can you write a function for this? it duplicates four times in the code 
> after that you can squash and merge it, also it’s good to add a TODO about creating a self-signed certificate if no cert and key provided.

sure, will work on it later today.